db/migrate/20231123111357_create_custom_field_sections.rb

```diff
@@ -28,9 +28,19 @@ def create_and_assign_default_section
       name: "Project attributes"
     )
 
-    # trigger acts_as_list callbacks via updating each record instead of bulk update
-    ProjectCustomField.find_each do |project_custom_field|
-      project_custom_field.update!(custom_field_section_id: section.id)
-    end
+    ActiveRecord::Base.connection.execute <<~SQL.squish
```

Comment:
We have the `execute_sql` helper which allows for passing automatically sanitized parameters. I don't think there is anything to worry about here in particular, but it's still a good habit to just use this precisely so that we don't have to think about potential issues with this. Besides this is otherwise surely going to turn up in some security analysis as a potential risk as well anyway and this can be avoided this way too.

So please use that instead like seen in the example in the source:

```
execute_sql "select id from users where mail = :email", email: params[:email]
```